### PR TITLE
Fixing broken links in Workers docs

### PR DESCRIPTION
**Pull request by deadlypants1973**
This is related to Jira issue: https://jira.cfops.it/browse/PCX-1813

### CODE REVIEW

File: products/workers/src/content/cli-wrangler/commands.md [hunk 1]
```diff
@@ -250,7 +250,7 @@ $ wrangler tail [--format $FORMAT] [--port $PORT] [--metrics-port $PORT]
   - The port for your local log server
 
 - `--metrics-port $PORT` <Type>int</Type>
-  - The port for serving [metrics information](https://developers.cloudflare.com/argo-tunnel/reference/arguments/#metrics) about the tunnel
```
Comment (marciocloudflare) [inline products/workers/src/content/cli-wrangler/commands.md:253]: Check if you're missing `.` in the end of the sentences here

File: products/workers/src/content/platform/limits.md [hunk 2]
```diff
@@ -158,7 +158,7 @@ There is no limit on the real runtime for a Workers script. As long as the clien
 
 Duration is the measurement of wall-clock time. This is measured in Gigabyte-seconds (GB-s). When a Worker is executed, it is allocated 128mb of [memory](/platform/limits#memory). As the Worker continues to execute that memory remains allocated, even during network IO requests.
 
-For example, when a Worker executes via a [scheduled event](/runtime-apis/scheduledevent), it executes for 4 seconds, including network-bound IO time: `4s x 0.125GB (or 128Mb) = .5 GB-s`.
+For example, when a Worker executes via a [scheduled event](/runtime-apis/scheduled-event), it executes for 4 seconds, including network-bound IO time: `4s x 0.125GB (or 128Mb) = .5 GB-s`.
```
Comment (marciocloudflare) [inline products/workers/src/content/platform/limits.md]: should be "four seconds"

**Review by marciocloudflare — APPROVED** [review]

File: products/workers/src/content/platform/limits.md (hunk 2 above)
Comment (deadlypants1973) [reply products/workers/src/content/platform/limits.md]: thank you, changed!

File: products/workers/src/content/cli-wrangler/commands.md (hunk 1 above)
Comment (deadlypants1973) [reply products/workers/src/content/cli-wrangler/commands.md:253]: thank you, resolved!

File: products/workers/src/content/cli-wrangler/configuration.md [hunk 3]
```diff
@@ -265,7 +265,7 @@ You can learn more about the standard patterns used for include and exclude in t
 
 Workers Sites projects use webpack by default. You can [bring your own webpack config](/cli-wrangler/webpack#using-with-workers-sites), however it is important to be cognizant of your `entry` and `context` settings.
 
-You can also use the `[build]` section with Workers Sites, as long as your build step will resolve dependencies in `node_modules`. See the [custom builds](configuration#build) section for more information.
+You can also use the `[build]` section with Workers Sites, as long as your build step will resolve dependencies in `node_modules`. See the [custom builds](/cli-wrangler/configuration#build) section for more information.
```
Comment (lukeed) [inline products/workers/src/content/cli-wrangler/configuration.md]: This particular link was just fixed by #1531 – section links on the same/current page can use just the `#section` for its link target.

```suggestion
You can also use the `[build]` section with Workers Sites, as long as your build step will resolve dependencies in `node_modules`. See the [custom builds](#build) section for more information.
```

**Review by lukeed — APPROVED** [review]

**Pull request merged**